packages/v4/patternfly-docs/content/accessibility/checkbox/checkbox.md

```diff
@@ -3,17 +3,78 @@ id: Checkbox
 section: components
 ---
 
-A **checkbox** is used to select a single item or multiple items, typically to choose elements to perform an action or to reflect a binary setting.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-Keyboard users will be able to interact with the checkbox component by pressing **Space** to select the input. We have added any attributes necessary to make it properly announced to screen readers.
+## Accessibility
 
-In general, the checkbox component already has accessibility built in. The **standalone input variant** will need special consideration.
+To implement an accessible PatternFly **checkbox** component:
 
-To make the checkbox **standalone input variant** accessible, add an `aria-label` with a label for the input. You can also use `aria-labelledby` to associate the checkbox with visible text on the page to explain the checkbox. 
+- Ensure the checkbox can be navigated to and interacted with via keyboard and other assistive technologies such as a screen reader
+- Ensure the checkbox can be toggled by clicking its visible text label, if it has one
+- Ensure the checkbox label has hover styles to indicate that it can be clicked
+- Ensure the checkbox input provides enough clickable space on its own
```

Comment:
I think it would be good to include some kind of standard/criteria for this size, and to make sure we're implementing it the same way, if we're going to include this. The issue we have to bump the size is pretty minor.

Reply:
Yeah that's a good point. I thought I had seen some mention about a preferred size in terms of a checkbox so will see if I can find it (though it may have been an unrelated component).

If not providing a standard for the size, we could possibly replace the verbiage with something along the lines of ensuring a checkbox isn't placed too closely to other interactive elements (though we may also want to include a standard spacing)

Reply:
@mcoker The PF site actually mentions [Success Criterion 2.5.5: Target Size](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) in the [Patternfly's accessibility](https://www.patternfly.org/v4/accessibility/patternflys-accessibility) page. Since there's the asterisk of, "WCAG 2.5.5 is included for reference only. This guideline suggests a size that is larger than what PatternFly requires", we could just omit verbiage about the input itself having enough clickable space for now. 

An answer for [Checkbox size of 'clickable' area on desktop](https://ux.stackexchange.com/a/124093) has some interesting examples regarding target size from other sources.